lib/Ryu/Async.pm

```diff
@@ -438,9 +440,15 @@ sub udp_server {
     my $port_f = $server->bind(
         service  => $uri->port // 0,
         socktype => 'dgram'
-    )->then(sub {
-        Future->done($server->write_handle->sockport)
-    });
+    )->then(
+        sub {
+            Future->done($server->write_handle->sockport);
+        },
+        sub {
+            my $err = "UDP server failed to bind to port " . ($uri->port // 0) . " - " . $_[0];
+            $log->errorf($err);
```

Comment:
Same here - probably isn't necessary to log, anything that cares about the error is likely to be handling it.